styletesla.css

```diff
@@ -12,6 +14,7 @@
 	align-items: center;
 }
 .navigation a {
+	font: 15px/18px montserratsemibold;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla-12.04/pullRequest?prid=5505629)